File: abc/abc080/a.cpp
// https://atcoder.jp/contests/abc080/submissions/30402208

#include <bits/stdc++.h>
#define all(x) (x).begin(), (x).end()
using namespace std;
typedef long long ll;

int main() {
  cin.tie(nullptr);
  ios_base::sync_with_stdio(false);

  int n, a, b; cin >> n >> a >> b;
  cout << min(a * n, b);
}

File: abc/abc080/a.java
// https://atcoder.jp/contests/abc080/submissions/43290787

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    int n = stdIn.nextInt();
    int a = stdIn.nextInt();
    int b = stdIn.nextInt();

    if(a * n < b) {
      System.out.println(a * n);
    } else {
      System.out.println(b);
    }
  }
}
